Formula/s/shuttle.rb

```diff
@@ -0,0 +1,31 @@
+class Shuttle < Formula
+  desc "Build & ship backends without writing any infrastructure files"
+  homepage "https://shuttle.dev"
+  url "https://github.com/shuttle-hq/shuttle/archive/refs/tags/v0.49.0.tar.gz"
+  sha256 "fe13c6a0717edd1d6ec838c6abf02d3230b379083d4daf8f63621d47d1ceded6"
+  license "Apache-2.0"
+  head "https://github.com/shuttle-hq/shuttle.git", branch: "main"
+
+  depends_on "pkgconf" => :build
+  depends_on "rust" => :build
+  depends_on "libgit2"
+
+  uses_from_macos "bzip2"
+
+  def install
+    ENV["LIBGIT2_NO_VENDOR"] = "1"
+    system "cargo", "install", *std_cargo_args(path: "cargo-shuttle")
+
+    generate_completions_from_executable(bin/"shuttle", "generate", "shell")
+    (man1/"shuttle.1").write Utils.safe_popen_read(bin/"shuttle", "generate", "manpage")
+
+    # cargo-shuttle is the same as shuttle
+    rm bin/"cargo-shuttle"
```

Comment:
this cannot be removed

<img width="728" alt="image" src="https://github.com/user-attachments/assets/adbc5b83-97bb-4e7c-8021-cfa87a10136f" />

also gonna update the formula to cargo-shuttle, like other package managers